airbyte-webapp/src/hooks/services/FormChangeTracker/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface FormChangeTrackerServiceApi {
+  trackFormChange: (id: string, changed: boolean) => void;
+  clearFormChange: (id: string) => void;
+  clearAllFormChanges: () => void;
+}
```

Comment:
I'd love to get @timroes opinion on this pattern.
I'm not the biggest fan and would prefer to place in the component or service, but content to fall either way.

Reply:
I don't have anything against this pattern. No strong feelings in either direction. Advantage of pulling it into a separate type file like this (as long as we need to export the type), is that we'd potentially prevent some cyclic dependency further down the road, in which case we'd then need to extract it into a separate type file.